src/domain/messages/helpers/message-verifier.helper.ts

```diff
@@ -163,6 +153,16 @@ export class MessageVerifierHelper {
       );
     }
 
+    if (
+      !this.isEthSignEnabled &&
+      signature.signatureType === SignatureType.EthSign
+    ) {
+      throw new HttpExceptionNoLog(
+        'eth_sign is disabled',
+        MessageVerifierHelper.StatusCode,
+      );
+    }
+
```

Comment:
This matches the pattern of other methods whereby we log unauthorized addreses as a priority.